app/indices/metric_index.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+ThinkingSphinx::Index.define(:metric, with: :real_time) do
+  indexes :friendly_name, as: :friendly_name
```

Comment:
I am thinking ...

We did not change before https://github.com/3scale/porta/blob/40771d47b8364c4721251740d1b4d933209e3624/config/thinking_sphinx.yml#L19-L21 and https://github.com/3scale/porta/blob/40771d47b8364c4721251740d1b4d933209e3624/config/docker/thinking_sphinx.yml#L18-L20 because we index in "real time" but we probably would like to do it for consistency.

Reply:
OK why aren't there more models included anyway? 🤔

Reply:
I think we can safely remove it because we are on thinking-sphinx v3

https://freelancing-gods.com/thinking-sphinx/v3/advanced_config.html#indexed-models